llm/migrations/0005_messagestore_type_alter_messagestore_message.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.6 on 2023-10-21 05:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("llm", "0004_embedding"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="messagestore",
+            name="type",
+            field=models.CharField(
+                choices=[("ai", "ai"), ("human", "human")],
+                default="human",
+                max_length=50,
+            ),
+        ),
```

Comment:
Excellent! This table structure (including the choices `ai`, `human`) was what Langchain code expects in order to work with multiple providers. So since we are just focusing on OpenAI, we can potentially simplify this a ton. Even possibly renaming table to `message` instead of `messagestore`.

```suggestion
        migrations.AddField(
            model_name="message",
            name="role",
            field=models.CharField(
                choices=[("system", "system"), ("assistant", "assistant"), ("user", "user")],
                default="user",
                max_length=50,
            ),
        ),
```

Reply:
I am going to let django create the migration file with those changes.